pkg/pluginmanager/manager.go

```diff
@@ -664,6 +627,59 @@ func InstallPluginsFromGroup(pluginName, groupIDAndVersion string, options ...Pl
 	return groupIDAndVersion, nil
 }
 
+// GetPluginsFromGroup fetches the plugins from the specified plugin group id and version.
+func GetPluginsFromGroup(groupIDAndVersion string, options ...PluginManagerOptions) (*plugininventory.PluginGroup, *plugininventory.PluginGroupIdentifier, error) {
```

Comment:
We can simplify this function by not returning the `*plugininventory.PluginGroupIdentifier`.  It turns out that all the information we need from that returned value is also included in the `*plugininventory.PluginGroup`